src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -65,7 +65,7 @@
     from transformers import PreTrainedTokenizer
     from vllm import CompletionOutput, RequestOutput
     from vllm.config import ModelConfig
-    from vllm.engine.protocol import AsyncEngineClient
+    from vllm.engine.protocol import EngineClient
```

Comment:
Since this was renamed in the latest changes, won't this break usage of the adapter with existing releases (which I thought @dtrifiro tries to maintain)?

Reply:
```suggestion
    try:
        from vllm.engine.protocol import EngineClient
    except ImportError:
        from vllm.engine.protocol import AsyncEngineClient as EngineClient
```
Maybe this is simple to alias like so?

Reply:
yep my bad, good one!